borg/archiver.py

```diff
@@ -997,7 +990,7 @@ def sig_info_handler(signum, stack):  # pragma: no cover
                 total = loc['st'].st_size
             except Exception:
                 pos, total = 0, 0
-            print("{0} {1}/{2}".format(path, format_file_size(pos), format_file_size(total)))
+            logger.warning("{0} {1}/{2}".format(path, format_file_size(pos), format_file_size(total)))
```

Comment:
well, i'm all ears. :) i have a solution that works here, how else are we going to handle those cases?